<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>go_to_line</class>
 <widget class="QDialog" name="go_to_line">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>252</width>
    <height>127</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Go to line</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Line you want to go to:</string>
       </property>
       <property name="buddy">
        <cstring>spinBox</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="ok_button">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancel_button">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>spinBox</tabstop>
  <tabstop>ok_button</tabstop>
  <tabstop>cancel_button</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
